site/gatsby-config.js

```diff
@@ -95,7 +95,10 @@ module.exports = {
     {
       resolve: "gatsby-plugin-sass",
       options: {
-        precision: 9,
+        implementation: require("node-sass"), // TODO: Move to dart-sass once webpack has native plugin support
```

Comment:
I believe dart-sass might be incompatible with some of our legacy codebases at the moment... last time we tried it caused build timeouts with no error messages 😬 Worth a shot again though, it'd be really nice to use it, but if we use it in Kaizen we need to be able to use it everywhere.